app/web/features/messages/requests/HostRequestStatusIcon.tsx

```diff
@@ -18,23 +18,21 @@ export default function HostRequestStatusIcon({
   let icon = null;
   let color = null;
 
-  const isRequestPast = dayjs(hostRequest.toDate).isBefore(dayjs().format("L"));
-
   if (s === HostRequestStatus.HOST_REQUEST_STATUS_ACCEPTED) {
     icon = <CheckIcon fontSize="inherit" />;
-    color = isRequestPast ? "grey.500" : "gray";
+    color = "green";
```

Comment:
```suggestion
    color = "gray";
```

I believe this one should be gray as it's accepted but not confirmed yet (sorry I realize that's confusing haha).